.circleci/config.yml

```diff
@@ -309,6 +309,7 @@ workflows:
       - api-local-test
       - api-build-test
       - ui-build-test
+      - api-bigquery-test
```

Comment:
Good question on integration tests. They could also probably run! 

I'm not too worried about the runtime thing, because its long enough that I have at least usually started doing something else while they are running. If we find its slow though/hard, we can always revert.

Reply:
Can folks who are outside our org make branches and push to them? Like can a thousand bot-controlled github accounts rack up millions of dollars in BigQuery expenses by repeatedly pushing?

Reply:
Anyone can create a fork, but only editors/owners can create branches. CI should only execute on branches within the repo - tested to verify my assertion: https://github.com/all-of-us/workbench/pull/2435